Gherkin Features/ID012_Sending_Friend_Requests.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Sending Friend Requests
```

Comment:
The scenarios sound more like they are responding to friend requests and not sending them